src/osw/core.py

```diff
@@ -30,6 +30,31 @@
 from osw.wiki_tools import SearchParam
 from osw.wtsite import WtPage, WtSite
 
+# Reusable type definitions
+OVERWRITE_OPTIONS = Union[
+    bool,  # False: never overwrite a property,
+    # True: always overwrite a property
+    Literal[
+        "only empty",  # only overwrite if the property is empty
```

Comment:
We should introduce an enum here, otherwise typos are likely

Reply:
Implemented with https://github.com/OpenSemanticLab/osw-python/commit/df2a469b850150d199e079609fff022080aa7195